app/models/shipit/pull_request.rb

```diff
@@ -104,6 +104,10 @@ def required_statuses
       end
     end
 
+    def merge_request?
+      !(merge_requested_by.nil? || merge_requested_at.nil?)
```

Comment:
Had you considered stating this in the affirmative?

```suggestion
      merge_requested_by.present? && merge_requested_at.present?
```